cli/parse_global_options_for_fuzzer.cpp

```diff
@@ -58,6 +73,14 @@ FuzzerArgs ParseGlobalOptionsForFuzzer(GlobalArgs &global_args, GlobalFuzzerOpti
         ("out_dir,o", 
             po::value<std::string>(&global_options.out_dir), 
             "Set output dir. Default is `/tmp/fuzzuf-out_dir`.")
+        ("executor,e",
+            po::value<fuzzuf::cli::ExecutorKind>(&global_options.executor)->default_value(global_options.executor),
+            "Specify fuzzing executor. Default is `native`.")
+        ("exec_path",
```

Comment:
`exec` がダブルミーニングになっていて気持ちわるいです。
`exec_path` は execut**or** path, 直後の `exec_timelimit_ms` は execution time limit、です。`executor_path` にしませんか？

Reply:
I don't think this naming makes confusion because it's clear that `exec` in `exec_path` is the executor as shown in the description. I favor the shorter option in the CLI in the CLI UI design aspect.

Reply:
Changed to use `proxy_path` in commit [7ca2faa](https://github.com/fuzzuf/fuzzuf/pull/47/commits/7ca2faa23c21a6d8a6110bdc74642f259bd65032).